ts/DataGrid/Table/Actions/RowsVirtualizer.ts

```diff
@@ -244,17 +256,30 @@ class RowsVirtualizer {
      */
     private renderRows(rowCursor: number): void {
         const { viewport: vp, buffer } = this;
-        const rowsPerPage = Math.ceil(
+        const isVirtualization = this.rowSettings?.virtualization;
+        const rowsPerPage = isVirtualization ? Math.ceil(
             vp.tbodyElement.offsetHeight / this.defaultRowHeight
-        );
+        ) : Infinity; // need to be refactored when add pagination
```

Comment:
Comments should begin with an uppercase character.
```suggestion
        ) : Infinity; // Need to be refactored when add pagination
```